packages/babel-plugin-component-annotate/README.md

```diff
@@ -47,6 +47,14 @@ Using pnpm:
 pnpm add @sentry/babel-plugin-component-annotate --save-dev
 ```
 
+## Options
+
+### `ignoreComponents`
```

Comment:
Super nit but I think my logic is sound: I think I would probably call it `ignore*d*Components`. `ignoreComponents` sounds like imperative and that it would accept a boolean rather than a list of components.